### PR TITLE
drupal-scaffold changes

### PR DESCRIPTION
**Pull request by unclegcb**
No issue created, just as discussed in tech-talk slack. Also included php default update here.

This wants PR #84 merged first, which is why you see noise from there.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -46,14 +45,14 @@
         "drupal/smart_trim": "^1.1",
         "drush/drush": "^10.0",
         "webflo/drupal-finder": "^1.2",
-        "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.1"
```
Comment (mariacha) [inline composer.json:50]: Does `zaporylie/composer-drupal-optimizations` not work with php 7.4 or something?

File: composer.json [hunk 2]
```diff
@@ -20,10 +20,9 @@
         }
     },
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
```
Comment (mariacha) [inline composer.json:23]: You'll want to update this in the pantheon.yml file and roll a new version of the docker instance circle uses to be fully ready to go.

**Review by mariacha — CHANGES_REQUESTED** [review]
Couple things I see without trying to pull down and build a new project.

File: composer.json (hunk 1 above)
Comment (unclegcb) [reply composer.json:50]: I had it in my head that we didn't need it anymore, but that's only true if we are on composer 2.0. Reverting!

File: composer.json (hunk 2 above)
Comment (unclegcb) [reply composer.json:23]: We have a working docker instance, but it includes screener. What do you think about just maintaining that one and not worrying about a separate, screener-free image?

Comment (mariacha) [reply composer.json:23]: I'm 100% fine with that.

File: composer.json [hunk 3]
```diff
@@ -46,14 +45,14 @@
...
-        "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.1"
+        "webmozart/path-util": "^2.3"
     },
     "require-dev": {
         "dmore/behat-chrome-extension": "^1.1",
         "drupal/coder": "^8.2",
         "drupal/config_suite": "^1.1",
         "drupal/console": "^1.9",
+        "drupal/core-composer-scaffold": "^8.8.1",
```
Comment (mariacha) [inline composer.json:55]: Hey, cool!

**Pull request merged**